uiowa/uiowa
# [Registrar] Add context if not default to academic calendar dates

Closes #8232 

# How to test

- `ddev blt ds --site=registrar.uiowa.edu`
- add the academic calendar block to a page, or try https://registrar.uiowa.ddev.site/calendars-and-deadlines/academic-calendar
- Set Category to 'Graduation and Commencement.'
- The 'Commencement RSVP Closed' events on November 20 should now have the associated college appended to the title, so the events are no longer identical
- You can also change the Category to 'Graduate Students' to see a few events with 'Graduate College' appended.
